samples/dialogs/src/main/java/com/badoo/ribs/samples/dialogs/rib/dialogs_example/DialogsPresenter.kt

```diff
@@ -79,10 +90,15 @@ internal class DialogsPresenterImpl(
 
     private val dialogEventConsumer: Consumer<Dialog.Event> = Consumer {
         when (it) {
-            Dialog.Event.Positive -> dummyViewInput.accept(DialogsView.ViewModel("Dialog - Positive clicked"))
-            Dialog.Event.Negative -> dummyViewInput.accept(DialogsView.ViewModel("Dialog - Negative clicked"))
-            Dialog.Event.Neutral -> dummyViewInput.accept(DialogsView.ViewModel("Dialog - Neutral clicked"))
-            Dialog.Event.Cancelled -> dummyViewInput.accept(DialogsView.ViewModel("Dialog - Cancelled"))
+            Dialog.Event.Positive -> view?.accept("Dialog - Positive clicked")
+            Dialog.Event.Negative -> view?.accept("Dialog - Negative clicked")
+            Dialog.Event.Neutral -> view?.accept("Dialog - Neutral clicked")
+            Dialog.Event.Cancelled -> view?.accept("Dialog - Cancelled")
```

Comment:
Instead of `view?` let's pass the view reference to this converted to a method. View reference should only be available inside the scope of `onViewCreated` + the methods it calls, otherwise it can quickly turn into an anti-pattern that leaks.